src/lib/api/ticket.ts

```diff
@@ -38,6 +38,30 @@ export async function create(
             throw new UnexpectedStatusCode(res.status);
     }
 }
+/** Edits the `title` field of a {@linkcode Priority}. Returns `false` if not found. */
+export async function editTicketDueDate(tid: Ticket['ticket_id'], due_date: Ticket['due_date']) {
+    const { status } = await fetch('/api/ticket/due_date', {
```

Comment:
You haven't implemented the server route for this yet. It would also be nice to rename it as `/api/ticket/due` instead.
```suggestion
    const { status } = await fetch('/api/ticket/due', {
```